tests/spectron/spectronWebDriver.js

```diff
@@ -275,5 +275,16 @@ class WebDriver {
      async close() {
         await this.driver.close();
     }
+
+    async sendMessagesAndVerifyToast(messages) {
+        for (var i = 0; i < messages.length; i++) {
+            await this.sendMessage(messages[i]).then(async()=>
```

Comment:
space after `()` and curly braces right after arrow notation

Reply:
fixed